cengage/react-magma
# fix: Update hex of secondaryColors

Issue: #975

## What I did


Update secondary colors, and remove all instances of the old one.

## Screenshots (if appropriate):
before:
![image](https://user-images.githubusercontent.com/91160746/206540774-702ec964-b469-4abb-ab05-67c64ba74bff.png)

after:
![image](https://user-images.githubusercontent.com/91160746/206565269-84424eeb-bcc7-4bb8-943c-6c1f2843a76b.png)



## Checklist 

## How to test

Confirm that the marketing button uses the correct new shade of yellow, and that all the states have the correct colors (solid/link, hover/active, inverse)
